### PR TITLE
Added Accessibility in Navbar Component

### LINKED ISSUE #2739: Navbar accessibility for keyboard users
**Current Behavior**
Keyboard users can't access the dropdown menu on the navbar. They can only be accessed when they are hover.

**Desired Behavior**
They should have have access to those drop down menus.

---
**Screenshots / Mockups**

The dropdown menu is only available to mouse users.

Screenshot (430)_LI

**Alternatives**

---

<h4>Contributor Resources</h4>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Palaksharma23**
**Description**

This PR fixes #2739

I have added keyboard accessibility to the Navbar 

 

https://user-images.githubusercontent.com/87171452/179660289-3ce7ea8f-376c-41a4-bd76-0ba155595c04.mp4

 **Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [✔ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Palaksharma23** [comment]
@leecalcote @adithyaakrishna  please once review this pull request.

**Comment by l5io** [comment]
🚀 Preview for commit 73475bbeee6262fe44a4ba7fd421d41df3e3f501 at: https://62d638df9107c341c60933fa--layer5.netlify.app

**Comment by adithyaakrishna** [comment]
@Palaksharma23 Thank you for working on this, do you mind fixing up the DCO? You can refer the deets here, https://github.com/layer5io/layer5/pull/2976/checks?check_run_id=7402090466

**Comment by l5io** [comment]
🚀 Preview for commit 6e1231a251376648912a28b623f233d4162a2e5d at: https://62d8040063025a56df460c2b--layer5.netlify.app

File: src/sections/General/Navigation/utility/ScrollspyMenu.js [hunk 1]
```diff
@@ -58,17 +101,17 @@ const ScrollspyMenu = ({ menuItems, ...props }) => {
...
+            <div className="nav-grid" ref={dropdowncontent}>
               <div className="hr">
-                <Link className="section" to={active.path}  >{active.name}
+                <Link className="section" onKeyDown={handlefocus} to={active.path}  >{active.name}
                 </Link>
                 {active.subItems.map((subItem, i) => (
                   <li key={i}>
                     { subItem.name === "Forum" ?
-                      <a href={subItem.path} target="_blank" className="sub-item" rel="noreferrer">
+                      <a href={subItem.path} target="_blank" className="sub-item" rel="noreferrer" onKeyDown={handlefocus}>
```
Comment (adithyaakrishna) [inline src/sections/General/Navigation/utility/ScrollspyMenu.js:111]: @Palaksharma23 What's your opinion on adding `aria-label` or `aria-describedby`?

This is a great reference for Accessible Anchor Links, https://amberwilson.co.uk/blog/are-your-anchor-links-accessible/ which might help you answer the above question :)

Comment (Palaksharma23) [reply src/sections/General/Navigation/utility/ScrollspyMenu.js:111]: Yes, you are right we should be adding these! 
I will commit the necessary changes :)

**Comment by adithyaakrishna** [comment]
@Palaksharma23 Any updates on this PR? 👀

**Comment by leecalcote** [comment]
@Palaksharma23... I can't quite say that this is really an issue that we're looking to solve. Likely, this will take a conversation. Are you able to join the next Websites call on Monday to discuss?

**Pull request closed without merging**